battleship.py

```diff
@@ -169,7 +175,12 @@ def isVertical(ship):
 Returns: bool
 '''
 def isHorizontal(ship):
-    return
+    if ship[0][0]==ship[1][0]==ship[2][0]:
+        ship.sort()
+        for col in ship:
+            if ship[0][1]+1==ship[1][1]==ship[2][1]-1:
+                return True
+    return False
 

```

Comment:
@mrudula-n  I have no problem with the logic you have to build. Try to re-think and make the code in fewer lines.